pom.xml

```diff
@@ -5,8 +5,9 @@
     <groupId>com.flowingcode.addons</groupId>
     <artifactId>font-awesome-iron-iconset</artifactId>
     <version>5.2.2-SNAPSHOT</version>
-    <name>FontAwesome Iron Iconset</name>
-    <description>Iron iconset based on FontAwesome</description>
+    <name>FontAwesome</name>
```

Comment:
Don't change name (at least for now, Vaadin Directory requires using the original `<name>` in all releases)